### ISSUE TITLE
pagination-nav <a> [aria-*] attributes do not match their roles

### ISSUE THREAD

**Issue by shadow81627** [bootstrap-vue/bootstrap-vue#5449]
### Describe the bug

I ran a lighthouse 6 audit on a page with b-pagination-nav and it came back with an aria error.
https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1590623485410-54343.report.html
`[aria-*] attributes do not match their roles`
Each ARIA `role` supports a specific subset of `aria-*` attributes. Mismatching these invalidates the `aria-*` attributes. [Learn more](https://web.dev/aria-input-field-name/?utm_source=lighthouse&utm_medium=cli).
```html
<a href="/recipes" aria-current="page" aria-label="Go to page 1" aria-posinset="1" aria-setsize="2" target="_self" class="page-link nuxt-link-exact-active active">1</a>
```

### Expected behavior

It seems like the <a> tag is missing a role attribute: https://www.w3.org/TR/wai-aria-1.1/#aria-setsize

### Versions

**Libraries:**

- BootstrapVue: 2.15.0
- Nuxt: 2.12.2

**Environment:**

- Device: Emulated Moto G4
- OS: Linux
- Browser: HeadlessChrome
- Version: 83.0.4103.61

**Comment by SatoshiRoppongi** [comment]
I have same problem. 
I think `a` element has to set `role=itemmenu` but it doesn't.
https://github.com/bootstrap-vue/bootstrap-vue/pull/4810/files#diff-704def8764e8ffb6961400f8a70971aeR459
It seems change above is degradation.
`role: isNav ? null : 'menuitem'`
I don't know the meaning of 'isNav' but I guess it's actually 
`role: isNav ? 'menuitem' : null`
No?
